packages/react/src/components/ErrorSummary/ErrorSummaryHeading.tsx

```diff
@@ -0,0 +1,40 @@
+import { Slot } from '@radix-ui/react-slot';
+import { useContext, useEffect } from 'react';
+
+import type { ListHeadingProps } from '../List';
+import { List } from '../List';
+
+import { ErrorSummaryContext } from './ErrorSummary';
+
+export type ErrorSummaryHeadingProps = {
+  /**
+   * Change the default rendered element for the one passed as a child, merging their props and behavior.
+   * @default false
+   */
+  asChild?: boolean;
+} & ListHeadingProps;
+
+export const ErrorSummaryHeading = ({
```

Comment:
Could this whole component not just be `export const ErrorSummaryHeading = List.Heading`? 😅  (also with new `displayName`)

Reply:
We need the id, so I don't think so?